dothething/dothething/Assets.xcassets/Placeholder.imageset/Contents.json

```diff
@@ -0,0 +1,21 @@
+{
+  "images" : [
+    {
+      "filename" : "obama.jpg",
+      "idiom" : "universal",
+      "scale" : "1x"
+    },
+    {
+      "idiom" : "universal",
+      "scale" : "2x"
+    },
+    {
+      "idiom" : "universal",
+      "scale" : "3x"
+    }
+  ],
+  "info" : {
+    "author" : "xcode",
+    "version" : 1
+  }
+}
```

Comment:
The changes in this file appear to be adding a new image asset named "Placeholder" with three different scale factors (1x, 2x, and 3x). This is a common practice for supporting different device screen resolutions in iOS apps. The changes look appropriate and follow the expected structure for an image asset catalog.